mesonbuild/wrap/wraptool.py

```diff
@@ -99,21 +99,36 @@ def get_latest_version(name: str, allow_insecure: bool) -> T.Tuple[str, str]:
     version, revision = latest_version.rsplit('-', 1)
     return version, revision
 
-def install(options: 'argparse.Namespace') -> None:
-    name = options.name
-    if not os.path.isdir('subprojects'):
-        raise SystemExit('Subprojects dir not found. Run this script in your source root directory.')
+def install_one(name: str, allow_insecure: bool) -> None:
     if os.path.isdir(os.path.join('subprojects', name)):
-        raise SystemExit('Subproject directory for this project already exists.')
+        print(f'Subproject directory for {name} already exists.')
+        return
     wrapfile = os.path.join('subprojects', name + '.wrap')
     if os.path.exists(wrapfile):
-        raise SystemExit('Wrap file already exists.')
-    (version, revision) = get_latest_version(name, options.allow_insecure)
-    url = open_wrapdburl(f'https://wrapdb.mesonbuild.com/v2/{name}_{version}-{revision}/{name}.wrap', options.allow_insecure, True)
+        print(f'Wrap file for {name} already exists.')
+        return
+    (version, revision) = get_latest_version(name, allow_insecure)
+    url = open_wrapdburl(f'https://wrapdb.mesonbuild.com/v2/{name}_{version}-{revision}/{name}.wrap', allow_insecure, True)
     with open(wrapfile, 'wb') as f:
         f.write(url.read())
     print(f'Installed {name} version {version} revision {revision}')
 
+def install(options: 'argparse.Namespace') -> None:
```

Comment:
```suggestion
def install(options: argparse.Namespace) -> None:
```

Reply:
Quotes are not needed because of the `from __future__ import annotations`. I tend to correct them when I saw them on a line I had to edit anyway.

Reply:
*technically* this line was not edited. It was moved downwards and code was refactored around it.

It is just that github's diff algorithm isn't detecting this line as the same. It may be possible to pass different algorithm options to the `git diff` commandline that cause it to detect the identical line better, though it's not super important one way or another.